include/VulkanInfoProvider.hpp

```diff
@@ -3,19 +3,20 @@
 #include <lug/Config.hpp>
 #include <lug/Graphics/Renderer.hpp>
 #include <lug/Graphics/Vulkan/Renderer.hpp>
+#include <lug/Graphics/Vulkan/API/RTTI/Enum.hpp>
 
 #if defined(LUG_SYSTEM_WINDOWS)
-    #pragma warning(push)
-    #pragma warning(disable : 4003)
+#pragma warning(push)
+#pragma warning(disable : 4003)
 #endif
 #include <json/json.hpp>
 #if defined(LUG_SYSTEM_WINDOWS)
-    #pragma warning(pop)
+#pragma warning(pop)
 #endif
 
 class VulkanInfoProvider {
```

Comment:
So, why do you need an instantiable class for just one method? Like I have said for `GPURequestor` and `Router` it's probably better to use a non-instantiable class.